Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -2501,6 +2502,16 @@ class X86Assembler {
         m_formatter.immediate8((uint8_t)laneIndex);
     }
 
+    void vpinsrb_mrr(uint8_t laneIndex, int offset, RegisterID base, XMMRegisterID xmm2, XMMRegisterID xmm1)
+    {
+        ASSERT(laneIndex < 16);
+        // https://www.felixcloutier.com/x86/pinsrb:pinsrd:pinsrq
+        // VEX.128.66.0F3A.W0 20 /r ib VPINSRB xmm1, xmm2, r32/m8, imm8
+        // B	NA	ModRM:reg (w)	VEX.vvvv (r)	ModRM:r/m (r)	imm8
+        m_formatter.vexNdsLigWigThreeByteOp(PRE_SSE_66, VexImpliedBytes::ThreeBytesOp3A, OP3_PINSRB, (RegisterID)xmm1, (RegisterID)xmm2, (RegisterID)base, offset);
+        m_formatter.immediate8(laneIndex);
+    }
+
```

Comment:
Place them in the right part in X86Assembler. This is not the place used for defining AVX functions.
Please strictly follow to the style in X86Assembler to keep consistency.